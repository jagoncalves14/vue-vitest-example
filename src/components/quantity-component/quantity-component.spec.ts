import { mount } from '@vue/test-utils'
import QuantityComponent from './quantity-component.vue'

describe('QuantityComponent', () => {
  describe('Mount', () => {
    it('should mount correctly', () => {
      const wrapper = mount(QuantityComponent)
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
