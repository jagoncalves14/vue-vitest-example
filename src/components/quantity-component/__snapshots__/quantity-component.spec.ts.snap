// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuantityComponent Mount should mount correctly 1`] = `<input class="quantity-component" type="text">`;
